device/core/devdoc/modules_events_and_messages_design.md

```diff
@@ -17,8 +17,19 @@ The following table shows which clients support which events & messages:
 
 | client type | `sendEvent` | `sendOutputEvent` | Messages | InputMessages |
```

Comment:
>Messages [](start = 50, length = 8)

suggestion: use the API name (on('message') / on('inputMessage')) to make it symmetrical with the send columns